Description: Don't build the binary, we just want the lib
Author: Carlos F. Sanz <carlos@sanz.dev>
Origin: <upstream|backport|vendor|other>, <URL, required except if Author is present>
Bug: <URL to the upstream bug report if any, implies patch has been forwarded, optional>
Bug-<Vendor>: <URL to the vendor bug report if any, optional>
Forwarded: <URL|no|not-needed, useless if you have a Bug field, optional>
Applied-Upstream: <version|URL|commit, identifies patches merged upstream, optional>
Reviewed-by: <name and email of a reviewer, optional>
Last-Update: 2021-12-22 <YYYY-MM-DD, last update of the meta-information, optional>
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: phf-generator/Cargo.toml
===================================================================
--- phf-generator.orig/Cargo.toml
+++ phf-generator/Cargo.toml
@@ -18,10 +18,6 @@ description = "PHF generation logic"
 license = "MIT"
 repository = "https://github.com/sfackler/rust-phf"
 
-[[bin]]
-name = "gen_hash_test"
-required-features = ["criterion"]
-
 [[bench]]
 name = "benches"
 harness = false
